source/objects/responseClass.m

```diff
@@ -268,9 +268,15 @@ function plotResponse(obj,bodyNum,comp)
             %     
             DOF = {'Surge','Sway','Heave','Roll','Pitch','Yaw'};
             t=obj.bodies(bodyNum).time;
+            if comp<4
             pos=obj.bodies(bodyNum).position(:,comp) - obj.bodies(bodyNum).position(1,comp);
```

Comment:
@salhus If we apply an initial linear displacement in the translational modes this will still cause the `pos` to be 0 at time = 0 and oscillate about the displacement value. Therefore, my suggestion is to subtract off the location of the center of gravity.

Reply:
@nathanmtom  Thanks for catching that.

I noticed that the bodies property, only stores the motion and forces. It doesn't have the cg information at  the moment.
Do you think I can modify that to have the cg information?

Reply:
Yes, you should be able to amend the function to pull in the body object to pull the center of gravity information.

Reply:
@nathanmtom  Thanks for the recommendation. I implemented it that way in the latest commit.

Reply:
@nathanmtom Sorry I deleted that comment when I realized I could do that, and didn't refresh the page to see your comment. But now i think it is doing just that.